axelrod/strategies/gambler.py

```diff
@@ -0,0 +1,111 @@
+from axelrod import Actions, Player, init_args, random_choice
+from itertools import product
+
+C, D = Actions.C, Actions.D
+
+
+class Gambler(Player):
+    """
+    A LookerUp class player which will select randomly an action in some cases.
+    It will always defect the last 2 turns.
+    Most comments and structure come from the LookerUp class.
+    New comments start with GK
```

Comment:
Could you move these last two lines outside of the docstring? (Perhaps an inline comment just below?) Only because the docstring will appear here: http://axelrod.readthedocs.org/en/latest/reference/all_strategies.html and without the whole code this line could be slightly confusing.